### PR TITLE
implement new cool-gray palette

### PR DESCRIPTION
**Pull request by spasovski**
I am sure it will need some more adjustments but it still feels quite close to the original.

### CODE REVIEW

File: public/static/global.css [hunk 1]
```diff
@@ -155,7 +163,7 @@
     --digital-blue-300: hsl(209, 85%, 75%);
     --digital-blue-400: hsl(209, 95%, 60%);
     --digital-blue-500: hsl(209, 100%, 39%);
-    --digital-blue-600: hsl(209, 100%, 34%);
+    --digital-blue-600: hsl(209, 77%, 46%);
```
Comment (hamilton) [inline public/static/global.css]: was this change intentional?

**Review by hamilton — CHANGES_REQUESTED** [review]
Thanks for getting this filed. This is a good start, but I think there are some differences here that feel pretty stark:

- the background color is noticeably darker in the app.
- I'm wondering if the color set is a bit too neutral. If you compare what's at the internal staging site to what's in this branch (specifically the search functionality), you can see the diff (see the screenvid). The linear version might be a better bet.
- we should probably jump on Zoom to talk about this more and screen share and figure out what we'll need to tweak in the usage of the colors.

Let me know how you want to proceed here. I think maybe the sine curve for saturation actually makes things not cooled enough.

Kapture 2019-12-31 at 8 58 33

File: public/static/global.css (hunk 1 above)
Comment (spasovski) [reply public/static/global.css]: nope, good catch
